spec/javascript/alchemy_admin/components/page_select.spec.js

```diff
@@ -0,0 +1,80 @@
+import { renderComponent } from "./component.helper"
+
+// import jquery and append it to the window object
+import jQuery from "jquery"
+window.$ = jQuery
```

Comment:
Do we need to add this in a `beforeEach` function to make this isolated?

I also think Jest has a `global.window` or `globalThis`